### PR TITLE
Added type info to matches display

### PR DESCRIPTION
**Pull request by Kraxxis**
Appended type information to the omnifunc display, following your suggestion in Issue #6 

2018-01-08-225026_681x311_scrot

(This is my first PR, so let me know if I'm doing something wrong, and I'll do my best to correct it)

### CODE REVIEW

File: autoload/asyncomplete/sources/flow.vim [hunk 1]
```diff
@@ -44,7 +44,7 @@ function! s:handler(opt, ctx, params, id, data, event) abort
         if a:data == 0
             let l:res = json_decode(a:params['stdout_buffer'])
             if !empty(l:res) && !empty(l:res['result'])
-                let l:matches = map(l:res['result'], '{"word": v:val["name"], "dup": 1, "icase": 1, "menu": "[Flow]"}')
+                let l:matches = map(l:res['result'], '{"word": v:val["name"], "dup": 1, "icase": 1, "menu": "[Flow]    " . v:val["type"]}')
```
Comment (prabirshrestha) [inline autoload/asyncomplete/sources/flow.vim]: we need to put this under a config value similar to `prefer_local`. Some people might not prefer this.

**Review by prabirshrestha — CHANGES_REQUESTED** [review]
Congrats on your first PR!

File: autoload/asyncomplete/sources/flow.vim (hunk 1 above)
Comment (Kraxxis) [reply autoload/asyncomplete/sources/flow.vim]: Sounds good, ill work on that addition tonight.

**Comment by prabirshrestha** [comment]
@Kraxxis looks good. Another feature would also be to always add the type info in preview window. refer to `:h completed-items`

```
info		more information about the item, can be displayed in a
		preview window
```

**Pull request merged**